### ISSUE TITLE
Delete *.A11yStrings.js files

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#795]
Once https://github.com/phetsims/rosetta/issues/193 is wrapped up. We should delete all a11y strings files. I currently see 19.

- [x] area-model-common (https://github.com/phetsims/area-model-common/issues/191)
- [x] balloons-and-static-electricity (https://github.com/phetsims/balloons-and-static-electricity/issues/463)
- [x] color-vision (https://github.com/phetsims/color-vision/issues/133)
- [x] coulombs-law (https://github.com/phetsims/coulombs-law/issues/117)
- [x] energy-forms-and-changes (https://github.com/phetsims/energy-forms-and-changes/issues/322)
- [x] faradays-law (https://github.com/phetsims/faradays-law/issues/156)
- [x] friction (completed, on a branch) (https://github.com/phetsims/friction/issues/187)
- [x] gravity-force-lab-basics (https://github.com/phetsims/gravity-force-lab-basics/issues/199)
- [x] gravity-force-lab (https://github.com/phetsims/gravity-force-lab/issues/263)
- [x] inverse-square-law-common (https://github.com/phetsims/inverse-square-law-common/issues/81)
- [x] john-travoltage (https://github.com/phetsims/john-travoltage/issues/345)
- [x] joist (https://github.com/phetsims/joist/issues/617)
- [x] molarity (https://github.com/phetsims/molarity/issues/205)
- [x] molecules-and-light (https://github.com/phetsims/molecules-and-light/issues/315)
- [x] ohms-law (https://github.com/phetsims/ohms-law/issues/157)
- [x] resistance-in-a-wire (https://github.com/phetsims/resistance-in-a-wire/issues/212)
- [x] rutherford-scattering (https://github.com/phetsims/rutherford-scattering/issues/167)
- [x] scenery-phet (https://github.com/phetsims/scenery-phet/issues/589)
- [x] sun (https://github.com/phetsims/sun/issues/584)

**Comment by zepumph** [comment]
I think this should be on hold until https://github.com/phetsims/chipper/issues/798 is completed.

**Comment by zepumph** [comment]
https://github.com/phetsims/molarity/issues/205 is the first sub issue we are doing with this, in part to make sure rosetta is ready for it.

**Comment by zepumph** [comment]
I wonder if we should hold off on this until es6-modules are worked out. Perhaps it doesn't make sense to put extra burden on that process (since a11y strings are so plentiful).

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/chipper/issues/798 (directional formatting issue) has a workaround and so it no longer blocks this issue.

**Comment by zepumph** [comment]
Right! I think though that there are two things that should keep this issue on hold:

* Waiting for molarity to be published to ensure that rosetta is set up and working well for this
* Perhaps knowing more about how strings will work with the new es6 modules, because I'm unsure about is we are even keeping the en.json files.

**Comment by zepumph** [comment]
https://github.com/phetsims/rosetta/issues/193 is wrapped up. Molarity was converted in https://github.com/phetsims/molarity/issues/205 and has been published with this. We are ready for this whenever. I will say though that I will likely wait until the es6 module conversion in a few weeks.

**Comment by zepumph** [comment]
Sub issues have been created. We will handle this for each individual repo from here. Closing.
